src/publisher.rs

```diff
@@ -276,6 +276,39 @@ pub extern "C" fn z_publisher_keyexpr(publisher: &z_loaned_publisher_t) -> &z_lo
     publisher.key_expr().transmute_handle()
 }
 
+/// Sets congestion control for the publisher
+#[no_mangle]
+#[allow(clippy::missing_safety_doc)]
```

Comment:
no need for #[allow(clippy::missing_safety_doc)], since the function is not unsafe